storage/src/mmr/bitmap.rs

```diff
@@ -147,13 +147,19 @@ impl<H: CHasher> Bitmap<H> {
         }
     }
 
-    /// Convert a bit offset into a bit mask.
-    fn mask_for(bit_offset: u64) -> u8 {
+    /// Convert a bit offset into a bit mask for the byte containing that bit.
+    pub(crate) fn mask_for(bit_offset: u64) -> u8 {
```

Comment:
nit: wonder if it would be clearer to call this `chunk_bit_mask` (better implying it is to be used within a chunk)?

Reply:
technically it's for a specific byte in a chunk. Changed to "chunk_byte_bit_mask", though maybe a bit wordy?